addons/web_editor/static/src/scss/wysiwyg_snippets.scss

```diff
@@ -1232,22 +1237,9 @@
         }
 
         // Color Pickers
-        .o_we_so_color_palette.o_we_user_value_widget {
-            &:first-of-type {
-                margin-right: $o-we-sidebar-content-field-multi-spacing * 4;
-            }
-
-            + .o_we_so_color_palette {
-                // Countering the margin of the first colorpicker and targeting
-                // the div to avoid misalignment of we-title (theme tab)
-                &:nth-of-type(2) {
-                    > div {
-                        margin-left: -$o-we-sidebar-content-field-multi-spacing * 4;
-                    }
-                    we-title + div {
-                        margin-left: 0;
-                    }
-                }
+        .o_we_so_color_palette.o_we_user_value_widget:not(.d-none) {
+            + .o_we_user_value_widget:not(.o_we_so_color_palette) {
+                margin-left: $o-we-sidebar-content-field-multi-spacing * 4 !important;
```

Comment:
Seems weird and in any case it is complicated: it should be commented. Feels strange to me: this targets widgets which are the immediate follower of a non-hidden colorpicker. Does that mean that in the case I have a visible colorpicker followed by a hidden colorpicker followed by a widget, I magically don't need that `margin-left`?

Also since you now added the `:not(.d-none)` part on the main selector, it means it impacts the following rules as well (`.o_we_color_preview {`...), it does not seems to be something we want. Doing "do not target invisible elements" should not be a thing.

Reply:
Thanks for the feedback, the rule was indeed non sense. It should be good now